system-addon/common/Reducers.jsm

```diff
@@ -177,28 +177,11 @@ function Sections(prevState = INITIAL_STATE.Sections, action) {
         }
         return section;
       });
-
-      // Invariant: Sections array sorted in increasing order of property `order`.
-      // If section doesn't exist in prevState, create a new section object. If
-      // the section has an order, insert it at the correct place in the array.
-      // Otherwise, prepend it and set the order to be minimal.
+      // Otherwise, append it
```

Comment:
I suppose @k88hudson is taking a look, but I believe for webextensions, we might want to prepend. Or insert after topsites. Or.. only worry about it when webextension stuff lands. ;)

Reply:
the order here isn't significant because we order them when rendering. On section register, we do prepend to the order pref. https://github.com/mozilla/activity-stream/pull/4022/files#diff-46c4b10ce28f1463cea556e4020e95f4R284

Reply:
But that's a good point. I am currently prepending to the top above topsites. But maybe we should check if the first slot is topsites then insert in the second slot?

Reply:
And I guess it doesn't matter until that stuff lands yeah 👍

Reply:
Oh nevermind, I see the prepending happening in `SectionsManager`.